### PR TITLE
Generate defaults in readme and index.js

### LINKED ISSUE #261: Keep defaults in README consistent with those from index.js
There are some inconsistencies between the defaults in the README and the index.js. 

```
rulesConfiguration.addRule('watchers-execution', [0, '$destroy']);

"angular/watchers-execution": [0, "$digest"],
```

The sections should be generated by the docs task like the rules section.

### PR DESCRIPTION
**Pull request by palortoff**
Fixes #261

### CODE REVIEW

**Comment by tilmanpotthof** [comment]
Thank for the PR. It cool that your used the new template mechanism and that it also works for the defaults section.

I'm not 100% sure, if the "truth" for the defaults should be defined in the documentation. Also, that the index.js is generated by a template feels a little dangerous.

My first idea when I wrote the issue was to use the index.js definition the generate the docs, but I forgot to mention this in the description.

What do you think @palortoff?

**Comment by Gillespie59** [comment]
I will prefer to use the default value in the index.js file for the generation of the documentation.

**Comment by palortoff** [comment]
Ok, valid points. I'll come up with another way of generating the defaults section.

**Comment by tilmanpotthof** [comment]
@palortoff I think it easy to transform your solution. I'd just import the index.js and replace the rule information from the doc annotation with those from the index.js. Do you want to finish the issue or should I transform your approach based on your code?

**Comment by tilmanpotthof** [comment]
@palortoff Are here any updates? Do you want to finish this feature or should I?

**Comment by tilmanpotthof** [comment]
@palortoff I merged the existing conflicts with the `development` branch. You can pull from the Gillespie59 repo now.

    git remote add Gillespie59 git@github.com:Gillespie59/eslint-plugin-angular.git
    git checkout issue-261_defaults 
   git pull Gillespie59 issue-261_defaults

**Comment by palortoff** [comment]
@tilmanpotthof 
Thanks for merging.
I hope I'll get it done this weekend.

**Comment by palortoff** [comment]
@tilmanpotthof 
done!
Index.js is not touched by docs task, defaults for readme are read from index.js.

I discarded the old branch and started from scratch. This way we have avoided a lot of reverting and polluting the history.

File: README.md [hunk 1]
```diff
@@ -145,12 +146,13 @@ Users may use the shareable [eslint-config-angular](https://github.com/dustinspe
         "angular/typecheck-object": 2,
         "angular/typecheck-regexp": 2,
         "angular/typecheck-string": 2,
-        "angular/watchers-execution": [0, "$digest"],
+        "angular/watchers-execution": [0, "$destroy"],
```
Comment (tilmanpotthof) [inline README.md]: The value in index.js is wrong and should be "$digest"

File: README.md [hunk 2]
```diff
@@ -118,25 +119,25 @@ Users may use the shareable [eslint-config-angular](https://github.com/dustinspe
...
         "angular/module-setter": 2,
         "angular/no-angular-mock": 0,
         "angular/no-controller": 0,
         "angular/no-cookiestore": 2,
-        "angular/no-digest": 2,
-        "angular/no-http-callback": 2,
-        "angular/no-inline-template": [0, {"allowSimple": true}],
+        "angular/no-digest": 0,
+        "angular/no-http-callback": 0,
+        "angular/no-inline-template": [0, {"allow-simple": true}],
```
Comment (tilmanpotthof) [inline README.md]: The key in index.js is wrong and should be "allowSimple"

**Comment by tilmanpotthof** [comment]
@palortoff Thanks for the changes. Looks good to me! There are two defaults that are already wrong in the `index.js`. I made inline comments for them.

**Comment by palortoff** [comment]
@tilmanpotthof 
changed the defaults in index.js

**Comment by tilmanpotthof** [comment]
Nice.

@Gillespie59 I think the PR is ready to be merged.

**Comment by tilmanpotthof** [comment]
@Gillespie59 0.15.0?

**Comment by Gillespie59** [comment]
Oops. I have already published the 0.15.0. Did not see your comment. Will be available with the 0.16.0 (that will publish maybe this week-end)

**Pull request merged**